stormlog/infer/cli.py

```
@@ -0,0 +1,274 @@
+    """Run the inference CLI."""
+    parser = build_parser()
+    args = parser.parse_args(argv)
+    if args.infer_command is None:
+        parser.print_help()
+        return 0
+    try:
+        if args.infer_command == "profile":
+            return cmd_profile(args)
+        if args.infer_command == "analyze":
+            return cmd_analyze(args)
+    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)